host/sgx/sgx_enclave_common_wrapper.c

```diff
@@ -113,6 +124,10 @@ static void _load_sgx_enclave_common_impl(void)
     if (_module)
     {
         OE_CHECK(_lookup_function("enclave_create", (void**)&_enclave_create));
+#ifndef OE_SKIP_CREATE_EX /* disabled until PSW update. */
```

Comment:
It would be better to avoid conditional compilation, and instead use runtime checks. The SDK ought to selectively enable/disable features at runtime based on what is available.

Reply:
Removed conditional compilation and made _lookup_function() failure to find enclave_create_ex() to a log.